api-extractor/src/definitions/ApiPackage.ts

```diff
@@ -7,18 +7,38 @@ import ApiEnum from './ApiEnum';
 import ApiFunction from './ApiFunction';
 import { IApiItemOptions } from './ApiItem';
 import ApiItemContainer from './ApiItemContainer';
+import TypeScriptHelpers from '../TypeScriptHelpers';
 
 /**
   * This class is part of the ApiItem abstract syntax tree.  It represents the top-level
   * exports for an Rush package.  This object acts as the root of the Analyzer's tree.
   */
 export default class ApiPackage extends ApiItemContainer {
-  constructor(analyzer: Analyzer, rootFileSymbol: ts.Symbol) {
-    super({
+  private static _getOptions(analyzer: Analyzer, rootFileSymbol: ts.Symbol): IApiItemOptions {
+    let statement: ts.VariableStatement;
+    let foundDescription: ts.Node = undefined;
+    for (const statementNode of (rootFileSymbol.declarations[0] as ts.SourceFile).statements) {
+      if (statementNode.kind === ts.SyntaxKind.VariableStatement) {
+        statement = statementNode as ts.VariableStatement;
```

Comment:
This typecast should be unnecessary.  Could take a quick look at why the tagged union isn't working?

https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#tagged-union-types
https://www.typescriptlang.org/docs/handbook/advanced-types.html